storage/redis/peer_store.go

```diff
@@ -84,34 +89,53 @@ func addKeyPrefix(namespacePrefix string, command string) string {
 	return command
```

Comment:
remove the function and directly concatenate the keyPrefix with "seeder" and "leecher"